src/main/asciidoc/_chapters/external_apis.adoc

```diff
@@ -341,7 +376,8 @@ curl -vi -X DELETE \
 
 |===
 
-.Endpoints for `Scan` Operations
+[[stateful.scanner.endpoints]]
+Stateful endpoints for scan operations
```

Comment:
should this be `.Stateful endpoints for scan operations` Currently not rendering as Table #
Same for other new table headers

Reply:
Nice catch, indeed, thanks. :+1: 
Will fix it.